### ISSUE TITLE
Is there any chance for Padme ability

### ISSUE THREAD

**Issue by zlyjestem** [punkUser/xwing_math#20]
It's hard (at least for me) to do math on attacks using padme ability - it would be great if you find time for that

**Comment by punkUser** [comment]
Yes, definitely on the short list to add. It's a bit tricky in terms of interactions with some other abilities like rebel Han pilot, but I agree it's hard to do in ones head so definitely something I want to add soon.
